modules/criteoBidAdapter.js

```diff
@@ -19,20 +20,241 @@ import { ajax } from '../src/ajax.js';
  */
 
 const GVLID = 91;
-export const ADAPTER_VERSION = 36;
+export const ADAPTER_VERSION = 37;
 const BIDDER_CODE = 'criteo';
-const CDB_ENDPOINT = 'https://bidder.criteo.com/cdb';
+const CDB_ENDPOINT = 'https://bidder.criteo.com/openrtb_2_5/pbjs/auction/request';
 const PROFILE_ID_INLINE = 207;
-export const PROFILE_ID_PUBLISHERTAG = 185;
 export const storage = getStorageManager({ bidderCode: BIDDER_CODE });
 const LOG_PREFIX = 'Criteo: ';
+const TRANSLATOR = ortb25Translator(true, Object.freeze([
+  ...DEFAULT_RULES,
+  moveRule('regs.gpp_sid'),
+  moveRule('device.sua')
```

Comment:
I think it makes sense to add these directly into the library's `EXT_PROMOTIONS` (they're missing because it was written when these fields didn't exist).

Reply:
Sure, done.